core/src/main/scala/org/apache/spark/api/r/RUtils.scala

```diff
@@ -23,6 +23,10 @@ import java.util.Arrays
 import org.apache.spark.{SparkEnv, SparkException}
 
 private[spark] object RUtils {
+  // Local path where R binary packages built from R source code contained in the spark
+  // packages specified with "--packages" or "--jars" command line option reside.
+  var rPackages: Option[String] = None
```

Comment:
This path is set only on localhost, won't change ever. This path is similar and a companion one to the local SparkR package path (<SPARK_HOME>/R/lib), will only be retrieved when running SparkR in client modes.